client/src/js/services/CashflowService.js

```diff
@@ -0,0 +1,24 @@
+(function (angular) {
```

Comment:
Could you clean up all these IIFEs?  Our `gulp` build process does it automatically.